vendor/modules.txt

```diff
@@ -715,3 +743,4 @@ sigs.k8s.io/yaml
 # k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.18.3
 # k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.18.3
 # k8s.io/sample-controller => k8s.io/sample-controller v0.18.3
+# volcano.sh/apis => volcano.sh/apis v1.3.0-k8s1.18.3-alpha.3
```

Comment:
could we remove the vendor directory, l think it is not necessary if we use go mod?

Reply:
> could we remove the vendor directory, l think it is not necessary if we use go mod?

Of course.

Reply:
we'd better keep vendor. It is convient and no need download dependent package everytime.